### PR TITLE
credit @Sc00bzT and clarify offer is still on

### PR DESCRIPTION
**Pull request by jacksingleton**

### CODE REVIEW

File: src/pages/security-contact-vulnerability-disclosure.php [hunk 1]
```diff
@@ -26,4 +26,9 @@
...
 someone tricks me into inserting the string "BackdoorPoCTwitter" into a release
 of any of my software projects, I will pay them $100 USD.
-</p>
+
+<p>
+First winner: @Sc00bzT (https://twitter.com/Sc00bzT/status/730903007014076416)
+<br/><br/>
+Offer still stands!
+</p>
(BackdoorPoCTwitter)
```
Comment (defuse) [inline src/pages/security-contact-vulnerability-disclosure.php]: Nice try!

Comment (defuse) [inline src/pages/security-contact-vulnerability-disclosure.php]: I'll accept the PR once this is removed. ;)

Comment (jacksingleton) [inline src/pages/security-contact-vulnerability-disclosure.php]: ah looks like github fixed their bug. this was hidden from the GH diff when I submitted it 👍

Comment (defuse) [inline src/pages/security-contact-vulnerability-disclosure.php]: Being completely lazy at reviewing PRs pays off!

**Pull request closed without merging**